test/MultibyteTest.php

```diff
@@ -27,76 +32,79 @@ public function multibyteEncodings()
         ];
     }
 
-    public function getXmlWithXXE()
+    public function getXmlWithXXE(): string
     {
         return <<<XML
-<?xml version="1.0" encoding="{ENCODING}"?>
-<!DOCTYPE methodCall [
-  <!ENTITY pocdata SYSTEM "file:///etc/passwd">
-]>
-<methodCall>
-    <methodName>retrieved: &pocdata;</methodName>
-</methodCall>
-XML;
+            <?xml version="1.0" encoding="{ENCODING}"?>
+            <!DOCTYPE methodCall [
+                <!ENTITY pocdata SYSTEM "file:///etc/passwd">
+            ]>
+            <methodCall>
+                <methodName>retrieved: &pocdata;</methodName>
+            </methodCall>
+            XML;
```

Comment:
Just a note, in case anybody wonders why I did this: PHP 7.3 adds the ability to indent heredocs and nowdocs. In each case, any leading whitespace before the closing marker is considered indentation, and stripped from each line of the string. This makes them far more readable when declared in indented code, which was the whole point of the change.